catalyst_voices/packages/catalyst_voices_blocs/lib/src/registration/cubits/keychain_creation_cubit.dart

```diff
@@ -38,6 +49,38 @@ final class KeychainCreationCubit extends Cubit<CreateKeychain> {
     }
   }
 
+  Future<void> downloadSeedPhrase() async {
+    final mnemonic = _seedPhraseState.seedPhrase?.mnemonic;
+    if (mnemonic == null) {
+      throw StateError('SeedPhrase is not generated. Make sure it exits first');
+    }
+
+    FutureOr<Uri> buildWebMnemonicDownloadUri() {
+      return Uri.dataFromString(
+        mnemonic,
+        encoding: utf8,
+        base64: true,
+      );
+    }
+
+    FutureOr<Uri> buildIOMnemonicDownloadUri() {
+      throw UnimplementedError();
+    }
+
+    final uri = CatalystPlatform.isWeb
+        ? await buildWebMnemonicDownloadUri()
+        : await buildIOMnemonicDownloadUri();
+
+    final path = Uri(path: 'catalyst_seed_phrase.txt');
+
+    try {
+      await _downloader.download(uri, path: path);
+    } catch (error, stackTrace) {
+      _logger.severe(error, stackTrace);
```

Comment:
![Screenshot 2024-09-30 at 19 10 03](https://github.com/user-attachments/assets/95493a28-af71-4e75-a532-7779b4a023de)

Just noticed it, also in my code, I'll add the message in registration_cubit, can you fix it here?